pkg/routine/config.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2020 by PACE Telematics GmbH. All rights reserved.
+// Created at 2020/02/27 by Marius Neugebauer
+
+package routine
```

Comment:
Also suggested was changing the package name to `background` to avoid confusion between goroutines and routines. And also calling the routines tasks or similar.

We should however keep the `routine` package to not break backwards compatibility.